roles/kubernetes/control-plane/templates/kubeadm-config.v1beta2.yaml.j2

```diff
@@ -18,8 +18,13 @@ nodeRegistration:
 {% endif %}
 {% if inventory_hostname in groups['kube_control_plane'] and inventory_hostname not in groups['kube_node'] %}
   taints:
+{% if kube_version is version('v1.20.0', '<') %}
```

Comment:
master min supported k8s version is 1.20, so you can (and should) drop the conditional part and keep only the key control-plan

Reply:
Done!